src/routes/webhooks/paddle.ts

```diff
@@ -405,6 +370,66 @@ const notifyNewPaddleTransaction = async ({
   await webhooks.transactions.send({ blocks });
 };
 
+export const processGiftedPayment = async (
+  data: TransactionCompletedEvent['data'],
+) => {
+  const con = await createOrGetConnection();
+  const { gifter_id, user_id } = data.customData as PaddleCustomData;
+
+  if (user_id === gifter_id) {
+    logger.error({ type: 'paddle' }, 'User and gifter are the same');
```

Comment:
maybe log user_id?

Reply:
I actually see we used to send `data` so probably want to return that

Reply:
Applied it on all three places.